daemon/protocol/python/scoot/client_lib.py

```diff
@@ -246,3 +292,17 @@ def domain(status):
                  exit_code=status.exit_code,
                  error=status.error)
   return [domain(x) for x in resp.status]
+
+
+def stop_daemon():
+  """ Request that the daemon be Stopped
+  """
+  global _client
+  if not is_started():
+    raise ScootException(Exception("Connection not started."))
+  req = daemon_pb2.EmptyStruct()
+  try:
+    _client.StopDaemon(req)
+  except Exception:
+      pass #  the call stops the daemon, but the grpc connection returns an error: swallow the error
```

Comment:
nit: python style guide suggests at least two spaces from statement and one single space after # for inline comments

https://www.python.org/dev/peps/pep-0008/#inline-comments